docs/src/pages/components/lists/lists.md

```diff
@@ -47,7 +47,8 @@ Below is an interactive demo that lets you explore the visual results of the dif
 
 ## Align list items
 
-You should change the list item alignment when displaying 3 lines or more, set the `alignItems` prop to "flex-start".
+You should change the list item alignment when displaying 3 lines or more.
+You can set the `alignItems="flex-start"` to do so.
```

Comment:
```suggestion
Use `alignItems="flex-start"` to do so.
```

Reply:
```suggestion
You can set the `alignItems="flex-start"` to fix the positioning.
```

Reply:
This is great feedback @mbrookes. I probably use "You can" instead of "Do" as well. This works as general advise similar to active vs. passive voice.

Though I would probably just merge these two sentences to avoid implicitness (and resolve https://github.com/mui-org/material-ui/pull/27730#discussion_r688817704 as well):

```suggestion
Use `alignItems="flex-start" when displaying three lines or more.`
```

But the "why" is still missing which also answers the question whether this is a SHOULD or MUST (in which case we may want to default to `flex-start` in the core implementation).

Reply:
I gave the problem a try. See the latest commit for evaluation.